Apps/Sandcastle/gallery/development/Shadows.html

```diff
@@ -144,6 +167,7 @@
 Cesium.knockout.getObservable(viewModel, 'cascades').subscribe(updateShadows);
 Cesium.knockout.getObservable(viewModel, 'lightSource').subscribe(updateShadows);
 Cesium.knockout.getObservable(viewModel, 'size').subscribe(updateSettings);
+Cesium.knockout.getObservable(viewModel, 'softShadows').subscribe(updateShadows);
```

Comment:
The latest shadows code that you merged in also added this line above, so you can remove this.